network/remove_peer.js

```diff
@@ -137,34 +139,66 @@ module.exports = (args, cbk) => {
         cbk);
       }],
 
-      // Check channels for peer to make sure that they can be cleanly closed
-      checkChannels: ['getChannels', ({getChannels}, cbk) => {
+      //Select channel to close if multiple are available
+      selectedChannel: ['getChannels', ({getChannels}, cbk) => {
         // Exit early when a peer is not specified or force closing is OK
         if (!args.public_key || !!args.is_forced) {
           return cbk();
-        }
+        } 
 
-        const selectedChannels = getChannels.channels.filter(channel => {
+        const channels = getChannels.channels.filter(channel => {
           // Ignore channels that are not the specified public key
           if (channel.partner_public_key !== args.public_key) {
             return false;
           }
 
-          // Exit early when there are no outpoints, consider all peer channels
-          if (!args.outpoints.length) {
-            return true;
-          }
+          //Return channels with the peer
+          return true;
+        });
 
-          // Only include selected channels
-          return args.outpoints.includes(asOutpoint(channel));
+        // Exit early when no channels are available
+        if (!channels.length) {
+          return cbk([404, 'NoChannelsToCloseWithPeer']);
+        }
+
+        //Exit early if not selecting a channel or only one is available
+        if (channels.length === 1 || !args.selectChannels) {
+          return cbk(null, channels);
```

Comment:
even if there is a single channel, it would probably be good to ask the user to select the channel

the reason being, if you are expecting a confirmation step here it might surprise you that the channel closes without asking you to select the channel